internal/v1/server_identity_test.go

```diff
@@ -18,8 +18,8 @@ func TestRedHatIdentity(t *testing.T) {
 
 	t.Run("VerifyIdentityHeaderMissing", func(t *testing.T) {
 		respStatusCode, body := tutils.GetResponseBody(t, srv.URL+"/api/image-builder/v1/version", nil)
-		require.Equal(t, http.StatusBadRequest, respStatusCode)
-		require.Contains(t, body, "missing x-rh-identity header")
+		require.Equal(t, http.StatusUnauthorized, respStatusCode)
+		require.Contains(t, body, "missing identity header")
```

Comment:
For requests without an identity header there is a slight change now, instead of 400 it returns 401 and the text is slightly different. These values are all hardcoded in the identity package functions `Extract` and `BasePolicy`. Functionally, everything should work as expected.